### ISSUE TITLE
headjs support

### ISSUE THREAD

**Issue by aaronchi** [mintdigital/asset_hat#6]
how about a headjs loader?

https://github.com/headjs/headjs

**Comment by rondevera** [comment]
Yep, thanks for the suggestion! Head JS, $script.js, and possibly others are contenders for an upcoming release.
